docs/nitro-deploy.md

```
@@ -1105,7 +1115,129 @@ Restart=always
 WantedBy=multi-user.target
 ```
 
-#### Tinfoil Router Inf4
+#### Tinfoil Router 0
+```sh
+sudo vim /etc/systemd/system/vsock-tinfoil-router-0.service
+```
+
+Add the following content:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)